sites/target.py

```diff
@@ -169,13 +169,18 @@ def submit_order(self):
         did_submit = False
 
         self.status_signal.emit(create_msg("Submitting Order", "normal"))
+        url = self.browser.current_url
         while not did_submit:
             try:
-                if len(self.browser.find_elements_by_id('creditCardInput-cvv')) > 0:
-                    self.browser.find_element_by_id('creditCardInput-cvv').send_keys(self.profile["card_cvv"])
-                self.browser.find_element_by_xpath('//button[@data-test= "placeOrderButton"]').click()
-                self.status_signal.emit(create_msg("Order Placed", "success"))
-                send_webhook("OP", "Target", self.profile["profile_name"], self.task_id, self.product_image)
-                did_submit = True
+                cvv_field = self.browser.find_elements_by_id('creditCardInput-cvv')
+                if cvv_field:
+                    if len(cvv_field[0].get_attribute('value')) == 0:
```

Comment:
What if cvv_field[0] does not exist?

Reply:
@aahurtado it will not reach that step unless the list returned by find_elements_by_id has at least one object in it, so index 0 will always exist there.